corehq/toggles/__init__.py

```diff
@@ -2600,13 +2600,28 @@ def _handle_attendance_tracking_role(domain, is_enabled):
     save_fn=_handle_attendance_tracking_role,
 )
 
+
+def _handle_geospatial_es_index(domain, is_enabled):
+    from corehq.apps.geospatial.tasks import index_es_docs_with_location_props
+    from corehq.apps.geospatial.utils import get_celery_task_tracker
+    from corehq.apps.geospatial.const import ES_INDEX_TASK_HELPER_BASE_KEY
+
+    if is_enabled:
+        celery_task_tracker = get_celery_task_tracker(domain, ES_INDEX_TASK_HELPER_BASE_KEY)
+        # This check simply enforces that each domain's task is handled in serial.
+        if not celery_task_tracker.is_active():
+            index_es_docs_with_location_props.delay(domain)
+
+
 MICROPLANNING = StaticToggle(
     'microplanning',
-    'Allows access to Microplanning GIS functionality',
+    'Allows access to GIS functionality',
```

Comment:
nit: This seems to be accidental during the merge with the master branch

Reply:
Ah yeah this definitely seems to be from a recent merge. Addressed in [eec9148](https://github.com/dimagi/commcare-hq/pull/35126/commits/eec91481cd8351db32f4ca40504f667b2d62b16c)